python/mxnet/ndarray/contrib.py

```diff
@@ -460,3 +461,65 @@ def _to_python_scalar(inputs, type_, name):
         return then_func()
     else:
         return else_func()
+
+def isinf(data, ctx=None):
+    """Checks whether a given NDArray has infinity or not
```

Comment:
the description should include the word element-wise. Something like, "Performs an element-wise check to determine if the NDArray contains an infinite element or not"